src/lib/get-api-version.js

```diff
@@ -18,7 +18,7 @@ const getApiVersion = async () => {
 };
 
 const getValidApiVersion = async () => {
-  return semver.valid(await getApiVersion());
+  return semver.valid(semver.coerce(await getApiVersion()));
```

Comment:
Isn't the coerce all that would be needed for the `4.5.0.6922454971` example you listed in https://github.com/medic/cht-conf/issues/596?

Reply:
`getValidApiVersion()` is being used elsewhere as well and valid and coerce return different types.